src/module/skb/bpf.rs

```diff
@@ -0,0 +1,81 @@
+//! Rust<>BPF types definitions for the SKB module. Fields are not translated in
+//! the BPF part and can be represented in various orders, depending from where
+//! they come from. Some handling might be needed in the unmarshalers.
+//!
+//! Please keep this file in sync with its BPF counterpart in bpf/skb_hook.bpf.c
+
+use std::mem;
+
+use anyhow::{bail, Result};
+use plain::Plain;
+
+use crate::{
+    core::events::{bpf::BpfRawSection, EventField},
+    event_field,
+};
+
+/// Helper to check a raw section validity and parse it into a structured type.
+fn parse_event<T>(raw_section: &BpfRawSection) -> Result<T>
```

Comment:
This helper is very convenient! Maybe move it to some common place so it can be used by other modules?